src/App.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import './App.scss';
+
+const goodsFromServer = [
+  'Dumplings',
+  'Carrot',
+  'Eggs',
+  'Ice cream',
+  'Apple',
+  'Bread',
+  'Fish',
+  'Honey',
+  'Jam',
+  'Garlic',
+];
+
+class App extends React.Component {
+  state = {
+    selectedGoods: ['Jam'],
+  }
+
+  selectItem = (item) => {
+    this.setState(state => ({
+      selectedGoods: [...state.selectedGoods, item],
+    }));
+  }
+
+  removeItem = (item) => {
+    this.setState(state => ({
+      selectedGoods: state.selectedGoods
+        .filter(currentItem => currentItem !== item),
+    }));
+  }
+
+  clearItems = () => {
+    this.setState({ selectedGoods: [] });
+  }
+
+  render() {
+    const { selectedGoods } = this.state;
+    const goodsIsSelected = selectedGoods.length > 0;
+
+    return (
+      <div className="app">
+        <h1>
+          Selected good: -
+          {goodsIsSelected
+            ? ` ${selectedGoods.slice(0, selectedGoods.length - 1).join(', ')}
+                ${selectedGoods.length > 1 ? 'and' : ''}
+                ${selectedGoods[selectedGoods.length - 1]}
+                ${selectedGoods.length === 1 ? 'is' : 'are'} selected`
+            : ' No goods selected'
+          }
+        </h1>
+        {goodsIsSelected
+          && <button type="button" onClick={this.clearItems}>X</button>
+        }
+        <ul className="app__list">
+          {goodsFromServer.map(item => (
+            <li
+              key={item}
+              className={`app__item ${selectedGoods.includes(item)
```

Comment:
you can create some variable (`isIncluded` for example) inside callback in `map` method before you return `<li>`, and use this variable in all places where you need it. with this approach, you can use the method includes only once